media-sound/audacity/audacity-9999.ebuild

```diff
@@ -23,32 +23,31 @@ else
 	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
 	MY_P="Audacity-${PV}"
 	S="${WORKDIR}/${PN}-sources-${PV}"
-	SRC_URI="https://github.com/audacity/audacity/releases/download/Audacity-${PV}/${PN}-sources-${PV}.tar.gz"
+	SRC_URI="
+		https://github.com/audacity/audacity/releases/download/Audacity-${PV}/${PN}-sources-${PV}.tar.gz
+		doc? ( https://github.com/audacity/audacity-manual/releases/download/v${PV}/audacity-manual-${PV}.tar.gz )
+	"
 fi
 
 SRC_URI+=" audiocom? ( ${MY_THREADPOOL} )"
 
-# GPL-2+, GPL-3 - Audacity itself
-# ZLIB - The ThreadPool single-header library
-# CC-BY-3.0 - Documentation
-LICENSE="GPL-2+
-	GPL-3
-	audiocom? ( ZLIB )
-"
+# Audacity, parts, documentation, ThreadPool
+LICENSE="GPL-3 GPL-2+ CC-BY-3.0 audiocom? ( ZLIB )"
 SLOT="0"
-IUSE="alsa audiocom ffmpeg +flac id3tag +ladspa +lv2 mpg123 +ogg
-	opus +portmixer sbsms test twolame vamp +vorbis wavpack"
+IUSE="alsa audiocom doc ffmpeg +flac id3tag +ladspa +lv2 mpg123
+	+ogg opus +portmixer sbsms test twolame vamp +vorbis wavpack"
+[[ ${PV} != 9999* ]] && USE+=" doc"
 REQUIRED_USE="
+	audiocom? ( wavpack )
 	opus? ( ogg )
+	test? ( mpg123 )
 	vorbis? ( ogg )
 "
 RESTRICT="!test? ( test )"
 
 # dev-db/sqlite:3 hard dependency.
 # dev-libs/glib:2, x11-libs/gtk+:3 hard dependency, from
 #   cmake-proxies/cmake-modules/dependencies/wxwidgets.cmake
-# sys-apps/util-linux hard dependency, from cmake-proxies/CMakeLists.txt
-#   for libuuid
```

Comment:
It's valid to have it there for libuuid especially if it hardcodes util-linux somehow and won't work on macOS.